### ISSUE TITLE
Invalid and incomplete instructions in README.md for using -Xasync

### ISSUE THREAD

**Issue by jdjohnston** [scala/scala-async#247]
Using the SBT example for -Xasync leads to SBT complaining that there is no `scalaOptions`. Makes sense, this is for the compiler. Changing it to `scalacOptions` then leads to scalac v2.13.3 giving the error "bad option: '-Xasync'". Perhaps this is only needed for Scala 2.12? Resolvable, but it certainly could be made easier for someone (like myself) new to scala-async.

That said, I look forward to seeing more from this interesting project

**Comment by SethTisue** [comment]
Scala 2.13.3 definitely accepts `-Xasync`, and yes it is required. I think you must be mistaken that you are using 2.13.3.

I fixed the `scalaOptions` vs `scalacOptions` thing in 7c494a71c6282ce3faabd0f299332667daca510c — good catch, thanks.

**Comment by jdjohnston** [comment]
Strange, when I commented out the -Xasync line in my build.sbt, it ran fine

`sbt:async> scalaVersion`
`[info] 2.13.2`

(Oops, 2.13.*3* was the command-line scalac, not the one sbt is using)

**Comment by SethTisue** [comment]
@retronym am I mistaken that `-Xasync` is required? what is the expected failure mode if it isn't supplied? might it appear to be "working", but without the async transform happening? if so, that seems like a trap

**Comment by SethTisue** [comment]
oh, you must be using the old 0.x plugin, which doesn't use `-Xasync`. try it with [1.0.0-M1](https://github.com/scala/scala-async/releases/tag/v1.0.0-M1)

@retronym mind updating the README to have the right advice about what version to use?

**Comment by jdjohnston** [comment]
OK. I changed my `build.sbt` to require Scala 2.13.3 (was using 2.13.2) and `scala-async` 1.0.0-M1 . Also re-enabled `-Xasync`. Minimal project compiled & ran fine.

BTW, while the snippets of code in `README.md` are helpful, I found `SmokeTest.scala` in the test directory to be of the most help in getting started. I have _some_ experience with concurrency & _some_ with Scala, but little with the combination.
